announcement/announcement-tool/tool/src/java/org/sakaiproject/announcement/tool/AnnouncementAction.java

```diff
@@ -1679,7 +1780,18 @@ protected String buildReviseAnnouncementContext(VelocityPortlet portlet, Context
 			AnnouncementActionState state, SessionState sstate)
 	{
 		context.put("service", contentHostingService);
-
+		String siteId = ToolManager.getCurrentPlacement().getContext(); // Esto es null busca otra forma. Buscar contexto-quiza buena opcion
```

Comment:
What is this comment about?

Reply:
Nothing, that's a commit from before it worked